internal/southboundhandler/handler.go

```diff
@@ -457,10 +472,17 @@ func extractBootstrapScript(secret *corev1.Secret, kind, providerID string) (str
 }
 
 // Determine the uninstall command from the bootstrap kind
-func getUninstall(_ string) (string, error) {
-	uninstall := ""
-	// uninstall is not supported; reprovision the host and create new cluster
-	return uninstall, nil
+func getUninstall(kind string) (string, error) {
+	switch kind {
+	case configTypeKubeadm:
```

Comment:
do we still need this provider? same question below for rke2

Reply:
We don't need it but when discussing this with Hyunsun we said we'd leave it in for now, it's not used right now.